packages/data-stores/src/launch/index.ts

```diff
@@ -10,7 +10,7 @@ export type { State };
 export type { LaunchStepType } from './types';
 export { STORE_KEY };
 
-use( plugins.persistence, persistOptions );
+registerPlugins();
```

Comment:
So many different ways this is implemented 🥲

Reply:
Yep...I also feel confused 🫠